src/algorithms/telemetry_decoder/gnuradio_blocks/sbas_l1_telemetry_decoder_gs.cc

```diff
@@ -82,6 +88,24 @@ sbas_l1_telemetry_decoder_gs::~sbas_l1_telemetry_decoder_gs()
                 {
                     LOG(WARNING) << "Exception in destructor closing the dump file " << ex.what();
                 }
+            if (pos == 0)
+                {
+                    if (!tlm_remove_file(d_dump_filename))
+                        {
+                            LOG(WARNING) << "Error deleting temporary file";
+                        }
```

Comment:
Is this necessary or desirable? If a dump file is configured (!!) then it should be produced, even if it is 0 bytes in length. It would be confusing to look for your dump file and not find it. If this is consistent with the rest of the code base, never mind. Consistency can be more important than correctness or rationality, when it comes to maintenance.